src/main/java/com/redhat/cloud/notifications/processors/webhooks/WebhookTypeProcessor.java

```diff
@@ -32,7 +31,12 @@
     private static final String TOKEN_HEADER = "X-Insight-Token";
 
     @Inject
-    Vertx vertx;
+    @SslVerificationEnabled
+    WebClient securedWebClient;
+
+    @Inject
+    @SslVerificationDisabled
+    WebClient unsecuredWebClient;
```

Comment:
How does this work? What mechanism ensures that this Webclient uses `no SslVerification` ? 
I see that the annotation is new (I though that it was something from vertx)

Reply:
This is a CDI injection with a qualifier. The magic happens in `WebClientProducer`. This producer can produce two `@Singleton` instances of `WebClient` depending on the qualifier used at the injection point:
- if `@SslVerificationDisabled` is present, then the injected `WebClient` will _trust_ all certificates
- if `@SslVerificationEnabled` is present, then the injected `WebClient` will _verifiy_ all certificates

Reply:
Ah OK, so it will match the qualifier used in the producer. Thanks!